dll/win32/shell32/brfolder.cpp

```diff
@@ -804,6 +805,84 @@ BrFolder_OnCommand(BrFolder *info, UINT id)
     }
 }
 
+static void
+GetTreeViewItemContextMenuPos(HWND hWnd, HTREEITEM hItem, POINT *ppt)
+{
+    RECT r;
```

Comment:
```suggestion
    RECT rc;
```
Don't use the one-letter variables. They are difficult to search.

Reply:
The entire function is 7 lines, only 4 of them relevant. How is this going to be difficult to search?

Reply:
Developers often search for target variables by greping variable names.
We live inside our brains. In perception, variable naming is important.

Reply:
You know grepping `'r'` (one-letter variable) is useless.

Reply:
Grepping for "rc" is not a lot better. This is in the realm of saying you want to grep for "int i".

Reply:
You should rename your name "w".

Reply:
Why not possible? Just rename it.

Reply:
You are not Elon Musk. Such a name is not acceptable.

Reply:
You are picking on things that don't matter. You are never going to search for this RECT nor its use.

Reply:
It seems like he's not being sincere. He's talented but sad.

Reply:
What the shit.

This is a tiny function, there is absolutely no difference between using `r` or `rc` in this context.

Reply:
Tiny function can become bigger.
The other developers are seeing this code.
Non-standard naming kills us.